integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -83,8 +96,18 @@ def get_config_from_frequency_sweep(full_df):
     parser = argparse.ArgumentParser()
     parser.add_argument('--const-config-path', required=False, default=None,
                         help='path containing existing ConstConfigIO configuration file')
+    parser.add_argument('--gpu-energy-margin', default=0, type=float, dest='gpu_energy_margin',
+                        help='Percentage of additional energy it is acceptable to consume if it results '
+                             'in a lower frequency selection for Fe.  This is useful for analyzing '
+                             'noisy systems that have many gpu frequencies near the Fe energy consumption value')
     parser.add_argument('--path', required=True,
                         help='path containing reports and machine.json')
+    parser.add_argument('--use-requested-frequency', action='store_true', default=False,
+                        dest='use_freq_req',
+                        help='Use the frequency that was requested during the frequency sweep instead '
+                             'of the achieved frequency for a given run.  This is useful in cases where '
+                             'multiple frequency domains or settings are impacted and the achieved '
+                             'frequency does not reflect the secondary impact')
```

Comment:
I'm not sure "secondary impact" has enough context in this help text. I think the following need to be clarified:
* _Impact on what?_ I think the answer to that question is F_e. That could be clarified by adding a description of the script as a whole to the `argparse.ArgumentParser()` call (e.g., something that says this generates a constconfig recommendation that finds the f_e point across frequency sweeps).
* This help text describes a secondary impact, so what's the primary impact? You might be able to explain that somewhere in this help text, but if the text feels like it's getting too long, it might be easier to avoid language that raises this question here (and instead leave it to the man page or some other/bigger document).